str/qc/qc_filters_associatr.py

```diff
@@ -0,0 +1,233 @@
+            ((mt.allele_1_minus_mode > 20) | (mt.allele_1_minus_mode < -30)),
+            hl.missing('float64'),
+            mt.LC,
+        ),
+        REPCI=hl.if_else(
+            ((mt.allele_1_minus_mode > 20) | (mt.allele_1_minus_mode < -30)),
+            hl.missing('str'),
+            mt.REPCI,
+        ),
+        REPCN=hl.if_else(
+            ((mt.allele_1_minus_mode > 20) | (mt.allele_1_minus_mode < -30)),
+            hl.missing('str'),
+            mt.REPCN,
+        ),
+        SO=hl.if_else(
+            ((mt.allele_1_minus_mode > 20) | (mt.allele_1_minus_mode < -30)),
+            hl.missing('str'),
+            mt.SO,
+        ),
+    )
+
+    mt = mt.annotate_entries(
+        GT=hl.if_else(
+            ((mt.allele_2_minus_mode > 20) | (mt.allele_2_minus_mode < -30)),
+            hl.missing('call'),
+            mt.GT,
+        ),
+        ADFL=hl.if_else(
+            ((mt.allele_2_minus_mode > 20) | (mt.allele_2_minus_mode < -30)),
+            hl.missing('str'),
+            mt.ADFL,
+        ),
+        ADIR=hl.if_else(
+            ((mt.allele_2_minus_mode > 20) | (mt.allele_2_minus_mode < -30)),
+            hl.missing('str'),
+            mt.ADIR,
+        ),
+        ADSP=hl.if_else(
+            ((mt.allele_2_minus_mode > 20) | (mt.allele_2_minus_mode < -30)),
+            hl.missing('str'),
+            mt.ADSP,
+        ),
+        LC=hl.if_else(
+            ((mt.allele_2_minus_mode > 20) | (mt.allele_2_minus_mode < -30)),
+            hl.missing('float64'),
+            mt.LC,
+        ),
+        REPCI=hl.if_else(
+            ((mt.allele_2_minus_mode > 20) | (mt.allele_2_minus_mode < -30)),
+            hl.missing('str'),
+            mt.REPCI,
+        ),
+        REPCN=hl.if_else(
+            ((mt.allele_2_minus_mode > 20) | (mt.allele_2_minus_mode < -30)),
+            hl.missing('str'),
+            mt.REPCN,
+        ),
+        SO=hl.if_else(
+            ((mt.allele_2_minus_mode > 20) | (mt.allele_2_minus_mode < -30)),
+            hl.missing('str'),
+            mt.SO,
+        ),
+    )
+
+    # calculate proportion of GTs that are defined per locus (after applying call-level filters, variant_qc.call_rate is not accurate anymore)
+    mt = mt.annotate_rows(
+        prop_GT_exists=hl.agg.count_where(hl.is_defined(mt.GT))
+        / (mt.variant_qc.n_called + mt.variant_qc.n_not_called)
+    )
+    # re-enforce locus level call rate >=0.9
+    mt = mt.filter_rows(mt.prop_GT_exists >= 0.9)
+
+    # wrangle mt, prepare for export_vcf():
+
+    # drop unneccessary columns prior to writing out
+    mt = mt.drop(
+        'allele_1_rep_length',
+        'allele_2_rep_length',
+        'allele_1_bp_length',
+        'allele_2_bp_length',
+        'allele_1_REPCI_over_CN',
+        'allele_2_REPCI_over_CN',
+        'allele_1_minus_mode',
+        'allele_2_minus_mode',
+        'allele_1_REPCI_size',
+        'allele_2_REPCI_size',
+        'allele_1_is_non_mode',
+        'allele_2_is_non_mode',
+    )
```

Comment:
writing to a VCF will drop everything except locus, alleles, quality, id, filter, format, and GT. Hail will drop everything else automatically